src/docs/environment/index.mdx

```diff
@@ -135,17 +175,24 @@ curl https://get.volta.sh | bash
 The volta installer will tell you to "open a new terminal to start using Volta", but you don't have to! You can just
 reload your shell:
 
-```shell
+```shell {tabTitle: Bash}
+exec "$SHELL"
+```
+
+```shell {tabTitle: Zsh}
 exec "$SHELL"
 ```
 
+```shell {tabTitle: Fish}
+exec $SHELL
+```
```

Comment:
Are you sure this is better since the three are exactly the same?

I can see *some* value if they ever were to differ.